src/components/AuthContextProvider.js

```diff
@@ -3,6 +3,7 @@ import { createContext, useContext, useState } from 'react';
 export const AuthContext = createContext();
 export const useAuthContext = () => useContext(AuthContext);
 import { login } from '../services/auth';
+import { userRegister } from '../services/reg';
```

Comment:
Rename to make it clearer :)

```suggestion
import { userRegister } from '../services/register';
```